### DISCUSSION TITLE
HELP!! Client Side Error

### DISCUSSION THREAD

**Discussion by WFMCCAdmin** [ChurchApps/ChurchAppsSupport#393] [B1] [answered]
I have tried to access the B1 church app using my personal account as an administrator for one of my groups. Every time I click the link to the group, I receive a server side error. I opened the source code and took pictures. I could not figure this out. I've tried on multiple computers using Chrome, Firefox, and Internet Explorer. All are updated.
<img width="440" alt="Capture2" src="https://github.com/user-attachments/assets/8be63192-da6f-4725-a7c1-0639b576e78e">
<img width="422" alt="Capture1" src="https://github.com/user-attachments/assets/b560e1db-14ef-4234-b08d-2e57b81e6b0f">

**Comment by vassbo** [comment, marked as answer]
@WFMCCAdmin This is most likely an issue on our side.
I see from the console that the `<MarkdownPreviewLight>` component is expecting a string, but getting `null`.

@jzongker I might be wrong, but I guess we can fix this by changing this line: https://github.com/ChurchApps/AppHelper/blob/1f77e558de716641cadcf845ef48c6c79e4f1945/src/components/markdownEditor/MarkdownPreviewLight.tsx#L42
to this: `const convertedText = getSpecialLinks(markdownString || "");`

**Comment by jzongker** [comment]
I'm really sorry guys.  I was just going through to close out old issues and noticed I overlooked this one! 

Yes, changing that line should fix the issue.  I've made the change.  @WFMCCAdmin let me know if you still encounter any problems.
